src/App.jsx

```diff
@@ -94,16 +129,19 @@ function App() {
   };
 
   const completeTodo = async (todoId) => {
```

Comment:
Something to consider. This function serves as a toggle for items because of your choice in UX design to have items crossed out instead of being removed. What a function does should be described by its name when possible . So `completeTodo` is a little misleading since it does two things but its name suggested at one action. Ideally, `completeTodo` should remove the list item from being rendered. No need to refactor or change your UX but this is an important distinction that should be considered going forward.